src/library/components/navigation/style.scss

```diff
@@ -0,0 +1,234 @@
+:global(#Activity_edit_container) {
+  :global(h3.menu) {
+    margin-top: 1em;
+    margin-bottom: 1em;
+    font-family: museo-sans, verdana, helvetica, arial, sans-serif;
+    font-weight: 300;
+  }
+  :global(.top-menu) {
+    background-color: #f5f5f5;
+    height: 80px;
+    padding-left: 0px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    :global(.left) {
+      height: 80px;
+      width: 250px;
+    }
```

Comment:
I'm concerned these global references are going to break the portal navigation in the interim period when we are still using the built in portal nav, and not full switched over to the portal-pages based nav.  I might just merge this anyway and see what breaks.

Reply:
Additionally if they are overriding styles defined in the portal this will be really confusing. So even if they don't break things, they shouldn't be defined here.